src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -528,6 +530,17 @@ internal static Matrix3x2 GetTransform(UIElement from, UIElement to)
 				return Matrix3x2.Identity;
 			}
 
+#if __SKIA__
+			Matrix4x4.Invert(to?.Visual?.TotalMatrix ?? Matrix4x4.Identity, out var invertedTotalMatrix);
+			var finalTransform = (from.Visual.TotalMatrix * invertedTotalMatrix).ToMatrix3x2();
```

Comment:
Can the `Visual` be `null` on skia? You are allowing it to be `null` for the `to` but not the `from`.

Reply:
No it can't, just a typo on my end.